src/cls/IPM/Main.cls

```diff
@@ -2232,8 +2263,13 @@ ClassMethod Install(ByRef pCommandInfo) [ Internal ]
             do ##class(%IPM.Lifecycle.Base).GetDefaultParameters(.tParams)
             merge tParams = pCommandInfo("data")
             set tParams("DeveloperMode") = $get(tParams("DeveloperMode"), 0)
-            set tParams("cmd") = "install"
-            set tParams("Install") = 1
+            set tParams("Force") = tForce
+            // In the case of an update, pCommandInfo has already been pre-populated with the necessary information.
+            // Therefore, tParams needs to be populated if this is an install command.
+            if '$get(tParams("Update"),0) {
+                set tParams("cmd") = "install"
```

Comment:
Need similar logic in load to not overwrite "cmd" subscript to "load" instead of "update"